.gitignore

```diff
@@ -198,4 +198,5 @@ $RECYCLE.BIN/
 /artifacts/
 /.vscode/
 /scripts/local/
-/x/incentives/keeper/osmosis_testing/
\ No newline at end of file
+/x/incentives/keeper/osmosis_testing/
+v7
```

Comment:
why are we gitignoring this?

Reply:
I am confused by that.

I added v7 beause of the protobufs temp folder there.

But I don't recall adding the other.  :/